sycl/include/CL/__spirv/spirv_ops.hpp

```diff
@@ -26,22 +26,23 @@
 
 #if (SYCL_EXT_ONEAPI_MATRIX_VERSION > 1)
```

Comment:
This is only changing the new API. This change should be added to the #else part as well.

Reply:
Thanks, that's in my todo list.